### PR TITLE
Gps test Setup, PoC

### PR DESCRIPTION
**Pull request by gast04**
Initial gps testing commit

### CODE REVIEW

File: tools/gpstest/src/LICENSE [hunk 1]
```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2018 OSQZSS
```
Comment (adeebshihadeh) [inline tools/gpstest/src/LICENSE:3]: gitignore this repo and add a script to clone it and do any other relevant setup

File: tools/gpstest/SConstruct [hunk 2]
```diff
@@ -0,0 +1,8 @@
+env = Environment(
+        CFLAGS = ["-O2"],
+        CC='gcc',
+    )
+
+src = ["src/limegps.c", "src/gpssim.c"]
+libs = ["LimeSuite", "pthread", "m"]
+env.Program("bin/LimeGPS", src, LIBS=libs, LIBPATH=['lib'])
```
Comment (adeebshihadeh) [inline tools/gpstest/SConstruct:8]: can't we just run `make`?

File: tools/gpstest/src/LICENSE (hunk 1 above)
Comment (adeebshihadeh) [reply tools/gpstest/src/LICENSE:3]: here's an example of something like this: https://github.com/commaai/openpilot/blob/master/selfdrive/debug/profiling/palanteer/setup.sh

File: tools/gpstest/SConstruct (hunk 2 above)
Comment (gast04) [reply tools/gpstest/SConstruct:8]: sure, but we use SConscript everywhere else^^

Comment (adeebshihadeh) [reply tools/gpstest/SConstruct:8]: that's only for our stuff. for example, what happens if gpssim.c gets renamed to gps_sim.c? `make` will still work, but this scons stuff will have to be updated

File: tools/gpstest/gpstest.sh [hunk 3]
```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+LimeGPS_BIN=LimeGPS/LimeGPS
+if test -f "$LimeGPS_BIN"; then
+    LD_PRELOAD=lib/libLimeSuite.so $LimeGPS_BIN $@
+else
+    echo "LimeGPS binary not found, run setup.sh first"
```
Comment (adeebshihadeh) [inline tools/gpstest/gpstest.sh:7]: always indent with two spaces

File: tools/gpstest/LimeUtil.sh [hunk 4]
```diff
@@ -0,0 +1,2 @@
+#!/bin/bash
+LD_PRELOAD=lib/libLimeSuite.so bin/LimeUtil $@
```
Comment (adeebshihadeh) [inline tools/gpstest/LimeUtil.sh:2]: why are these checked in? we generally don't check in binaries unless there's a good reason. do we just need to clone and build a separate project?

Comment (gast04) [reply tools/gpstest/LimeUtil.sh]: the LimeSuite repo is bigger, so I thought makes sense to prebuilt it once, but changed

**Pull request merged**